src/mumble/LookConfig.h

```diff
@@ -36,7 +36,7 @@ public slots:
 
 private:
 	/// Reload themes combobox and select given configuredStyle in it
-	void reloadThemes(const boost::optional< ThemeInfo::StyleInfo > configuredStyle);
+    void reloadThemes(const std::optional< ThemeInfo::StyleInfo > configuredStyle);
```

Comment:
```suggestion
    void reloadThemes(std::optional< ThemeInfo::StyleInfo > configuredStyle);
```
(or take by const-ref but just const doesn't make sense)